packages/create-svelte/package.json

```diff
@@ -22,8 +22,7 @@
 	"bin": "./bin.js",
 	"main": "./index.js",
 	"dependencies": {
-		"@clack/prompts": "^0.7.0",
-		"kleur": "^4.1.5"
+		"@clack/prompts": "^0.7.0"
 	},
 	"devDependencies": {
 		"@playwright/test": "^1.44.1",
```

Comment:
Do we need to add the engines block and @types/node to this, and other packages where `styleText` is used?